lang/src/org/partiql/lang/ast/StatementToExprNode.kt

```diff
@@ -232,6 +234,18 @@ private class StatementTransformer(val ion: IonSystem) {
             is JoinType.Full -> JoinOp.OUTER
         }
 
+    private fun Let.toLetSource(): LetSource {
+        return LetSource(
+            this.letBindings.map {
+                LetBinding(
+                    it.expr.toExprNode(),
+                    it.name.toSymbolicName()!!
```

Comment:
Pretty sure that `!!` is not needed here.

Reply:
Removed the not-null assertion op. Revised `toSymbolicName` to not use nullable guards (`?`)